### ISSUE TITLE
Why does deltaf_loader calculates temperature instead of using initial conditions

### ISSUE THREAD

**Issue by Quickrix** [Warwick-Plasma/epoch#367]
deltaf method implementation use two different approaches to get the temperature of background plasma:
1. deltaf_loader (init routine): calculates local temperature via call to params_local()
2. particle pusher: directly uses initial conditions (species%initial_conditions%temp_back) in f0()

In both cases, the background distribution weights are calculated using the same formula.

More over deltaf_loader() debug part contains comparison with f0() from pusher.

To put it simply, deltaf_loader calculates the temperature, which, by definition, should be equal to the initial conditions.

Is there some hidden meaning of doing like that?
Why doesn't deltaf_loader() use the simpler and lighter approach of using initial conditions?
